README.md

```diff
@@ -58,7 +58,22 @@ flake8 --select=DOC <FILE_OR_FOLDER>
 If you don't include `--select=DOC` in your command, _flake8_ will also run
 other built-in _flake8_ linters on your code.
 
-### 2.3. Native vs _flake8_
+### 2.3. As a pre-commit hook
+
+`pydoclint` is configured for [pre-commit](https://pre-commit.com/)
+and can be set up as a hook with the following `.pre-commit-config.yaml`
+configuration:
+
+```yaml
+- repo: https://github.com/jsh9/pydoclint
+  rev: 0.0.7
```

Comment:
If we put `0.0.7` here, do we need to update this number here every time we publish a new release?  I wonder what other pre-commit hooks do in their README.

Reply:
There are multiple ways to go about this
- Use a branch name like `main` as rev
- Use a generic non working term as template like `<latest_tag>`
- Use renovate to generate update PRs with new version ([example config for a nodejs hook](https://github.com/renovatebot/pre-commit-hooks/blob/main/renovate.json))

Reply:
Thanks!  I think the 2nd option is cleaner. I'll push a code change.